File: C++11/백준/Bronze/5063. TGN/TGN.cc
#include <iostream>
using namespace std;

int main(void){
    int t;
    cin >> t;
    for (int i=0;i<t;i++){
        int a,b,c;
        cin >> a >> b >> c;
        if (a<b-c){
            cout << "advertise" << endl;
        }
        else if (a==b-c){
            cout << "does not matter" << endl;
        }
        else{
            cout << "do not advertise" << endl;
        }
    }
}

File: C#/백준/Bronze/5063. TGN/TGN.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int t = int.Parse(Console.ReadLine());
            for (int i=0;i<t;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int a = aa[0]; int b = aa[1]; int c = aa[2];
                if(a<b-c){
                    Console.WriteLine("advertise");
                }
                else if(a==b-c){
                    Console.WriteLine("does not matter");
                }
                else{
                    Console.WriteLine("do not advertise");
                }
            }
        }
    }
}